app/controllers/sessions_controller.rb

```diff
@@ -6,16 +6,26 @@ def new
   def create
     user = User.find_by(email: params[:session][:email].downcase)
     if user && user.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user_url(user)
-    else
-      flash[:danger] = 'Invalid email/password combination'
-      render 'new'
+      if user.activated? 
+        log_in user
+        params[:session][:remember_me] == '1' ? remember(user) : forget(user)
```

Comment:
Yeah...considering that I am not initializing the remember token/digest to anything, forget does not do anything...thanks.